NewsApp/Controllers/NewsViewController/Cell/CountrySelectionView.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  CountrySelectionView.swift
+//  NewsApp
+//
+//  Created by Reuben Simphiwe Kuse on 2024/08/29.
+//
+
+import Foundation
+import UIKit
+
+class CountrySelectionView: UIView {
+    
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        print("CountrySelectionView Frame: \(self.frame)")
+    }
+    
+    // Closure that will be triggered when the view is tapped
+    var onTap: (() -> Void)?
```

Comment:
You could name this closure as `onTapToChangeCountry`.  Just an idea and your choice.

Reply:
okay noted @mhlangagc